### ISSUE TITLE
invalid syntax style 

### ISSUE THREAD

**Issue by Czaki** [brisvag/napari-gruvbox#1]
When try to run napari with installed `napari-gruvbox` it fails with following error:

```
napari          
Traceback (most recent call last):
  File "/home/czaki/.pyenv/versions/napari/bin/napari", line 8, in <module>
    sys.exit(main())
  File "/home/czaki/Projekty/napari/napari/__main__.py", line 570, in main
    _run()
  File "/home/czaki/Projekty/napari/napari/__main__.py", line 330, in _run
    viewer = Viewer()
  File "/home/czaki/Projekty/napari/napari/viewer.py", line 65, in __init__
    _initialize_plugins()
  File "/home/czaki/Projekty/napari/napari/plugins/__init__.py", line 65, in _initialize_plugins
    _install_npe2_themes()
  File "/home/czaki/Projekty/napari/napari/utils/theme.py", line 388, in _install_npe2_themes
    register_theme(theme.id, theme_dict, manifest.name)
  File "/home/czaki/Projekty/napari/napari/utils/theme.py", line 264, in register_theme
    theme = Theme(**theme)
  File "/home/czaki/Projekty/napari/napari/utils/events/evented_model.py", line 200, in __init__
    super().__init__(**kwargs)
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for Theme
syntax_style
  Incorrect `syntax_style` value provided. Please use one of the following:  default, emacs, friendly, colorful, autumn, murphy, manni, monokai, perldoc, pastie, borland, trac, native, fruity, bw, vim, vs, tango, rrt, xcode, igor, paraiso-light, paraiso-dark, lovelace, algol, algol_nu, arduino, rainbow_dash, abap, solarized-dark, solarized-light, sas, stata, stata-light, stata-dark, inkpot (type=assertion_error)      
```

**Comment by brisvag** [comment]
Uhm, this is not happening on my installation. `gruvbox-dark` and `gruvbox-light` are both available as pygments styles... I guess I have some other package installed?

**Comment by Czaki** [comment]
Ok. I found. 

gruvbox theme was added in https://github.com/pygments/pygments/pull/1763

That was released in pygments in 2.9, where napari requires pygments >= 2.6, so it looks like adding pygments >= 2.9 to requirements of this package.
